tasks.py

```diff
@@ -14,15 +12,28 @@
 ROOT_PATH = os.path.dirname(__file__)
 
 
-# TODO make these tasks namespaced
-# release = Collection(common.tasks.prepare, common.tasks.release)
+namespace = Collection()
+namespace.add_collection(
+    Collection(
+        common.tasks.prepare,
+        common.tasks.release,
+    ),
+    name='deploy',
+)
```

Comment:
I believe this will change the docs for deploying no? So it swill be `invoke deploy.release` instead of `invoke release`?

Reply:
Good point!

-ops PR at https://github.com/rtfd/readthedocs-ops/pull/447

-corporate-ops PR at https://github.com/rtfd/readthedocs-corporate-ops/pull/138